float32 x_pos
float32 y_pos
float32 steer_rot
float32 theta
float32 delta_yaw
float32 steer_rot_degree